### PR TITLE
Update php examples for management api

### PR DESCRIPTION
**Pull request by erjohnson**

### CODE REVIEW

File: articles/libraries/auth0-php/management-api.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ The `Management` class stores access to endpoints as properties of its instances
 This endpoint is documented [here](/api/management/v2#!/Users/get_users).
 
 ```php
-$results = $mgmt_api->users->search([
+$results = $mgmt_api->users()->getAll([
```
Comment (lbalmaceda) [inline articles/libraries/auth0-php/management-api.md:58]: @joshcanhelp can you bless the changes on *this* file, please?

Comment (joshcanhelp) [reply articles/libraries/auth0-php/management-api.md:58]: ✅

File: articles/libraries/auth0-php/management-api.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ if (! empty($results)) {
 This endpoint is documented [here](/api/management/v2#!/Clients/get_clients).
 
 ```php
-$results = $mgmt_api->clients->getAll();
+$results = $mgmt_api->clients()->getAll();
```
Comment (joshcanhelp) [inline articles/libraries/auth0-php/management-api.md:80]: ✅

**Review by lbalmaceda — APPROVED** [review]
Looks good. Thanks

**Pull request merged**